src/cclib/parser/gaussianparser.py

```diff
@@ -1730,6 +1723,48 @@ def extract(self, inputfile, line):
                 polarizability = utils.symmetrize(polarizability, use_triangle='lower')
                 self.polarizabilities.append(polarizability)
 
+        # IRC Computation convergence checks.
+        # -------- Sample extract for converged step --------
+        # IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC
+        # Pt  1 Step number   1 out of a maximum of  50
+        # Modified Bulirsch-Stoer Extrapolation Cycles:
+        #    EPS =    0.000010000000000
+        # Maximum DWI energy   std dev =  0.000000595 at pt     1
+        # Maximum DWI gradient std dev =  0.135684493 at pt     2
+        # CORRECTOR INTEGRATION CONVERGENCE:
+        #   Recorrection delta-x convergence threshold:    0.010000
+        #   Delta-x Convergence Met
+        # Point Number:   1          Path Number:   1
+        #   CHANGE IN THE REACTION COORDINATE =    0.16730
+        #   NET REACTION COORDINATE UP TO THIS POINT =    0.16730
+        #  # OF POINTS ALONG THE PATH =   1
+        #  # OF STEPS =   1
+        #
+        # Calculating another point on the path.
+        # IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC
+        # -------- Sample extract for unconverged intermediate step --------
+        # IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC
+        # Error in corrector energy =          -0.0000457166
+        # Magnitude of corrector gradient =     0.0140183779
+        # Magnitude of analytic gradient =      0.0144021969
+        # Magnitude of difference =             0.0078709968
+        # Angle between gradients (degrees)=   32.1199
+        # Pt 40 Step number   2 out of a maximum of  20
+        # Modified Bulirsch-Stoer Extrapolation Cycles:
+        #    EPS =    0.000010000000000
+        # Maximum DWI energy   std dev =  0.000007300 at pt    31
+        # Maximum DWI gradient std dev =  0.085197906 at pt    59
+        # CORRECTOR INTEGRATION CONVERGENCE:
+        #   Recorrection delta-x convergence threshold:    0.010000
+        #   Delta-x Convergence NOT Met
+        # IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC-IRC
+
+        if line[3:22] == "Delta-x Convergence":
+            if line[23:30] == "NOT Met":
+                self.optstatus[-1] = data.ccData.OPT_UNCONVERGED
+            elif line[23:26] == "Met":
+                self.optstatus[-1] = data.ccData.OPT_DONE
```

Comment:
Is there any else action?

Reply:
Not that I can think of... it is actually only changing the statuswhen necessary.